packages/core/src/helpers/value.ts

```diff
@@ -0,0 +1,126 @@
+import type { ParsedValue } from '../types';
```

Comment:
Maybe `globalValue` belongs here too.

Reply:
`globalValue` is checking for string values that start with `st-global`, I think I would place it in some `helpers/global`, because `helpers/value` contain helpers that deal with CSS value AST, but I'm only moving things as I'm refactoring code that relate to them. So not yet.